packages/powersync_attachments_helper/lib/src/attachments_queue.dart

```diff
@@ -33,14 +33,18 @@ abstract class AbstractAttachmentQueue {
   /// Return true if you want to ignore attachment
   Future<bool> Function(Attachment attachment, Object exception)? onUploadError;
 
+  /// Interval in minutes to periodically delete archived attachments
```

Comment:
Sorry I missed changing this comment. This sets the timer for `startPeriodicSync` which deletesArchived attachments and runs a sync again in case one of the attachments which was archived is now available to download.